playbook.py

```diff
@@ -0,0 +1,486 @@
+    phantom.debug('close_case_2() called')
+
+    #phantom.debug('Action: {0} {1}'.format(action['name'], ('SUCCEEDED' if success else 'FAILED')))
+
+    # collect data for 'close_case_2' call
+    results_data_1 = phantom.collect2(container=container, datapath=['create_case_1:action_result.data.*.number', 'create_case_1:action_result.parameter.context.artifact_id'], action_results=results)
+
+    parameters = []
+
+    # build parameters list for 'close_case_2' call
+    for results_item_1 in results_data_1:
+        if results_item_1[0]:
+            parameters.append({
+                'case_number': results_item_1[0],
+                # context (artifact id) is added to associate results with the artifact
+                'context': {'artifact_id': results_item_1[1]},
+            })
+
+    phantom.act(action="close case", parameters=parameters, assets=['partner'], name="close_case_2", parent_action=action)
+
+    return
+
+def hunt_file_1(action=None, success=None, container=None, results=None, handle=None, filtered_artifacts=None, filtered_results=None, custom_function=None, **kwargs):
+    phantom.debug('hunt_file_1() called')
+
+    #phantom.debug('Action: {0} {1}'.format(action['name'], ('SUCCEEDED' if success else 'FAILED')))
+
+    # collect data for 'hunt_file_1' call
+    inputs_data_1 = phantom.collect2(container=container, datapath=['add_case_event_1:artifact:*.cef.fileHashMd5', 'add_case_event_1:artifact:*.cef.fname', 'add_case_event_1:artifact:*.id'], action_results=results)
+
+    parameters = []
+
+    # build parameters list for 'hunt_file_1' call
+    for inputs_item_1 in inputs_data_1:
+        if inputs_item_1[0]:
+            parameters.append({
+                'hash': inputs_item_1[0],
+                'file_name': inputs_item_1[1],
+                # context (artifact id) is added to associate results with the artifact
+                'context': {'artifact_id': inputs_item_1[2]},
+            })
+
+    phantom.act(action="hunt file", parameters=parameters, assets=['partner'], callback=format_1, name="hunt_file_1", parent_action=action)
+
+    return
+
+def send_email_1(action=None, success=None, container=None, results=None, handle=None, filtered_artifacts=None, filtered_results=None, custom_function=None, **kwargs):
+    phantom.debug('send_email_1() called')
+
+    #phantom.debug('Action: {0} {1}'.format(action['name'], ('SUCCEEDED' if success else 'FAILED')))
+
+    # collect data for 'send_email_1' call
+    results_data_1 = phantom.collect2(container=container, datapath=['get_alert_details_1:action_result.data.*.name', 'get_alert_details_1:action_result.parameter.context.artifact_id'], action_results=results)
+    formatted_data_1 = phantom.get_format_data(name='format_1')
+
+    parameters = []
+
+    # build parameters list for 'send_email_1' call
+    for results_item_1 in results_data_1:
+        parameters.append({
+            'from': "Code42 Alert Response",
+            'to': "peter.briggs@code42.com",
```

Comment:
Yo email is in here

Reply:
I think it is not in the one in `playbooks/` though, so this file was probably unintentionally left

Reply:
Yep that playbook file has been removed